torch/quantization/observer.py

```diff
@@ -364,8 +371,8 @@ def __init__(self, dtype=torch.quint8, qscheme=torch.per_tensor_affine,
                                              reduce_range=reduce_range,
                                              quant_min=quant_min,
                                              quant_max=quant_max)
-        self.register_buffer('min_val', torch.tensor([]))
-        self.register_buffer('max_val', torch.tensor([]))
+        self.register_buffer('min_val', torch.tensor(float('inf')))
```

Comment:
Can we serialize tensors with inf/-inf correctly?

Reply:
Seems ok

```
>>> m
EmbeddingBag(1, 2, mode=mean)
>>> m.register_buffer("v", torch.tensor([float('inf')]))
>>> m.v
tensor([inf])
>>> ms = torch.jit.script(m)
>>> ms
RecursiveScriptModule(original_name=EmbeddingBag)
>>> ms.v
tensor([inf])
>>> torch.jit.save(ms, "/tmp/test.pt")
>>> ms_2 = torch.jit.load("/tmp/test.pt")
>>> ms_2.v
tensor([inf])
```